### PR TITLE
Update joining-a-testnet.md

### PR DESCRIPTION
**Pull request by jeremiegrr**

### CODE REVIEW

File: docs/docs/joining-a-testnet.md [hunk 1]
```diff
@@ -1,56 +1,19 @@
...
 ```
 
 You can also verify that you are running the correct version for the [network](../networks/).
 
 ```bash
 fetchd version
-fetchcli version
-```
-
-### Configuring the client `fetchcli`
```
Comment (daeMOn63) [inline docs/docs/joining-a-testnet.md:36]: we still need to configure the client, it's only renamed to  `fetchd` and dropped the `trust-node` config. 
So we can still set `chain-id` and `node`  options.

> Just FYI, some extra infos that don't really belong here, but we might want to have it on some other pages:


This config command avoid having to provide these flags on every other command
ie without config:
```
fetchd query staking validators --node https://rpc-stargateworld.fetch.ai:443 --chain-id stargateworld-1 
fetchd tx bank send account1 account2 1atestfet --node https://rpc-stargateworld.fetch.ai:443 --chain-id stargateworld-1
```
now with config:

```bash
fetchd config node https://rpc-stargateworld.fetch.ai:443
fetchd config chain-id stargateworld-1

# now the following will still work the same as above:

fetchd query staking validators 
fetchd tx bank send account1 account2 1atestfet 
```

And few extra bits:
```bash
fetchd config # display current config
cat ~/.fetchd/config/client.toml # this is the file where the config is written
```

**Review by daeMOn63 — CHANGES_REQUESTED** [review]
Looks good! Just a little section to revert, we'll still need it to keep every other commands simple

**Review by jeremiegrr — COMMENTED** [review]
for review

File: docs/docs/joining-a-testnet.md [hunk 2]
```diff
@@ -1,58 +1,53 @@
...
-
-To connect to the beacon world testnet users would simply need to run the following command
-
-	docker run -e MONIKER=my-first-fetch-node -e NETWORK=beaconworld fetchai/fetchd:0.5
+In order to join the test network you will need to have the correct version of the fetchd ledger available on your system. 
 
 ## Using a local version
 
-Assuming that you have following the [installation guide](../building/). You should now have `fetchd` and `fetchcli` successfully installed in your path. You can check this with the following command:
+Assuming that you have following the [installation guide](../building/). You should now have `fetchd` successfully installed in your path. You can check this with the following command:
```
Comment (daeMOn63) [inline docs/docs/joining-a-testnet.md:7]:
```suggestion
Assuming that you have followed the [installation guide](../building/). You should now have `fetchd` successfully installed in your path. You can check this with the following command:
```

File: docs/docs/joining-a-testnet.md [hunk 3]
```diff
@@ -1,58 +1,53 @@
...
-**Beacon World Example**
+In the case of the Stargate network this would be as follows:
 
-In the case of the beacon world network this would be as follows:
+```bash
+fetchd config chain-id stargateworld-1
+fetchd config node https://rpc-stargateworld.fetch.ai:443
+```
+  
+# now the following will still work the same as above:
```
Comment (daeMOn63) [inline docs/docs/joining-a-testnet.md:36]:
```suggestion
```

File: docs/docs/joining-a-testnet.md [hunk 4]
```diff
@@ -1,58 +1,53 @@
...
 ```
 
+And few extra bits:
+
+```bash
+fetchd config # display current config
+cat ~/.fetchd/config/client.toml # this is the file where the config is written
+```
+
+
```
Comment (daeMOn63) [inline docs/docs/joining-a-testnet.md:50]:
```suggestion
```
This doesn't really belong to "joining a testnet", more on a general CLI usage document

**Review by daeMOn63 — CHANGES_REQUESTED** [review]

**Comment by daeMOn63** [comment]
fixed and merged in #118

**Pull request closed without merging**